server/monitor.go

```diff
@@ -784,7 +784,7 @@ type RouteInfo struct {
 	Idle         string             `json:"idle"`
 	Import       *SubjectPermission `json:"import,omitempty"`
 	Export       *SubjectPermission `json:"export,omitempty"`
-	Pending      int                `json:"pending_size"`
+	Pending      int64              `json:"pending_size"`
```

Comment:
Can we make it `json:"pending_bytes"`, to match other monitoring endpoints?

Reply:
I think we leave as is already, also why changing to int64?

Reply:
Changed type because `r.out.pb` on line 835 is int64.

Sorry, I did not understand you reply.
If we have the same entity referred to by two different names, it is bad. If this name was not actually used, then can we fix it here?

Reply:
would rather not change the type and leave it as an `int` like ConnInfo, also if we leave as is then old code using the same structs would work already, like the natscli which imports these structs as is from the server.

Reply:
@alexbozhenko Type changes are breaking changes for exported types in the package API, so we can't do that.